### PR TITLE
[ticket/16221] Fix ACP statistics layout

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16221

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16221

### CODE REVIEW

**Comment by 3D-I** [comment]
immagine

immagine

immagine

immagine

File: phpBB/adm/style/acp_main.html [hunk 1]
```diff
@@ -98,109 +98,113 @@ <h3>{L_ERROR_MBSTRING_HTTP_OUTPUT}</h3>
 			<table class="table1 zebra-table no-header" data-no-responsive-header="true">
 				<thead>
 					<tr>
-						<th>{L_STATISTIC}</th>
-						<th>{L_VALUE}</th>
+						<th>{{ lang('STATISTIC}') }}</th>
```
Comment (vinny) [inline phpBB/adm/style/acp_main.html]: {{ lang('STATISTIC') }}

File: phpBB/adm/style/admin.css [hunk 2]
```diff
@@ -728,6 +728,18 @@ td {
 	border-collapse: separate;
 }
 
+.table2 {
+	display: inline-block;
```
Comment (hanakin) [inline phpBB/adm/style/admin.css]: this leads to a weird display issue on very widescreens above 1400px where the div and table are not the same width. Try `display: table` Have not checked on windows browsers where the original issue was with firefox i believe.

File: phpBB/adm/style/acp_main.html [hunk 3]
```diff
@@ -95,112 +95,116 @@ <h3>{L_ERROR_MBSTRING_HTTP_OUTPUT}</h3>
...
+						<th>{{ lang('STATISTIC') }}</th>
+						<th>{{ lang('VALUE') }}</th>
 					</tr>
 				</thead>
+
 				<tbody>
 					<tr>
-						<td class="tabled">{L_NUMBER_POSTS}{L_COLON}</td>
-						<td class="tabled"><strong>{TOTAL_POSTS}</strong></td>
+						<td class="tabled">{{ lang('BOARD_STARTED') ~ lang('COLON') }}</td>
```
Comment (hanakin) [inline phpBB/adm/style/acp_main.html]: why you are switching the order of two tables around? This makes for a lot of unnecessary diffs

Comment (3D-I) [reply phpBB/adm/style/acp_main.html]: I brought back the original arrangement of the elements, as in fact they are currently in 3.1.x / 3.2.x. The arrangement changed in https://github.com/phpbb/phpbb/pull/5187 and nobody noticed it.

---
3.1.x

immagine
---

3.2.x

immagine

File: phpBB/adm/style/admin.css (hunk 2 above)
Comment (3D-I) [reply phpBB/adm/style/admin.css]: I see now what you mean, I have a Full-HD monitor.
Using Firefox / Windows here BTW. The change you proposed doesn't do the trick.

Let me see if I can fix that glitch...

**Comment by 3D-I** [comment]
> 
> 
> why you are switching the order of everything around? This makes for a lot of unnecessary diffs

The arrangement changed in https://github.com/phpbb/phpbb/pull/5187 and nobody noticed it.
immagine

---

I brought back the original arrangement of the elements, as in fact they are currently in 3.1.x / 3.2.x. 

---
3.1.x

immagine
---

3.2.x

immagine

---
Current PR

immagine

File: phpBB/adm/style/admin.css (hunk 2 above)
Comment (3D-I) [reply phpBB/adm/style/admin.css]: Fixed with the latest commit, here's a grab (1920 pixels)

immagine

**Comment by hanakin** [comment]
change seems to work for all the mac browsers... but the original issue was with firefox on windows 10 which I am unable to test att. pending that works @marc1706 it looks good. Up to you if you want it in 3.3

**Comment by 3D-I** [comment]
Using Firefox on Windows here.

**Pull request merged**